tests/ui/binding/issue-53114-safety-checks.stderr

```diff
@@ -4,7 +4,7 @@ error[E0793]: reference to packed field is unaligned
 LL |     let _ = &p.b;
    |             ^^^^
    |
-   = note: packed structs are only aligned by one byte, and many modern architectures penalize unaligned field accesses
+   = note: this field is only 1-byte aligned, but its type requires more alignment
```

Comment:
So if I have a `packed(8)` struct with a field whose type cannot be determined, the error will say
> this field is only 8-byte aligned, but its type requires more alignment

That's not correct though -- the type *may* require higher alignment, but we don't actually know that for sure.

Reply:
Oh, you're right. It's a different concern than the field requiring an unknown alignment, but that is correct. Sorry for making the change without discussing it, thanks for your patience!